.travis.yml

```diff
@@ -27,13 +32,20 @@ install:
   - source activate test-environment
   # Ensure we will install all pip packages within conda
   - conda install pip
+  # enables sending of code coverage reports with the codecov command
+  - pip install codecov
   # Install numpy first to avoid headaches
   - pip install numpy
   # Install the r_packages
   - conda install rpy2
   - ./install_r_packages.sh
+  # build extensions
+  - pip install cython
+  - python setup.py build_ext
   # Install the packages needed for testing
-  - pip install .['test']
+  - pip install -e .['test']
```

Comment:
I'm not thrilled about this way of installing things since the code should install via `pip install  .`, including whatever needs to work for cython.  I read a comment you left on another issue that said this is the only way you can get it to work.  This is also how I installed things when I was developing the code, so perhaps I made a mistake somewhere which precludes the correct installation procedure (without `-e`).  

Really, we should fix the installation procedure to work for a simple `pip install .`, without `python setup.py build_ext` and `pip install -e`, instead of building this hiccup into the continuous integration.